### DISCUSSION TITLE
Building Better Explorer for older CPUs.

### DISCUSSION THREAD

**Discussion by RetroSoftwareRepository** [Gainedge/BetterExplorer#147] [Q&A] [answered]
Can I build Better Explorer for older CPUs like the Pentium MMX or Pentium 3? I'm looking to add some usability to an older system that I'm running Thin PC on that I feel could benefit from the enhanced File Explorer that Better Explorer provides, considering that later builds of Windows 8 (after Milestone 1, build 7850) won't even boot on my system (HAL related).

**Comment by DimitarCC** [comment, marked as answer]
No. Better Explorer currently works only on Windows 10 and newer. There are code limitations for older systems.

**Reply by RetroSoftwareRepository** [reply]
Shame... although I was hoping an older build from let's say 2016 could be tweaked to run on older Windows systems. Windows 8 and newer already have the Ribbon UI which makes Better Explorer redundant for the most part.

**Reply by DimitarCC** [reply]
Well old version of Better Explorer from 2016 may work on older systems with Windows 7 but that means porting latest changes to older version which is quite alot of work.
